<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>278</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Pending_Layout">
       <item>
        <widget class="QFrame" name="Pending">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item alignment="Qt::AlignLeft">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Pending Proposals</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="DatePublished_Pending">
            <property name="text">
             <string>Date Published</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QTableWidget" name="PendingTable">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>100</y>
        <width>278</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_Events_3">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_3">
          <widget class="QFrame" name="Events_Upcoming">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>276</width>
             <height>549</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="pushButton_8">
                 <property name="text">
                  <string>Upcoming Events</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_9">
                 <property name="text">
                  <string>Rescheduled Events</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_3">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_3">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <widget class="QFrame" name="Events_Upcoming_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>276</width>
             <height>549</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="pushButton_11">
                 <property name="text">
                  <string>Upcoming Events</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_12">
                 <property name="text">
                  <string>Rescheduled Events</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_4">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_4">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Event Approval</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>278</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_Proposals">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_3">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_5">
          <widget class="QFrame" name="UpcomingEvents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>276</width>
             <height>549</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="pushButton_17">
                 <property name="text">
                  <string>Approved Proposals</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_18">
                 <property name="text">
                  <string>Rejected Proposals</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_6">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_6">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_6">
          <widget class="QFrame" name="RejectedProposals">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>276</width>
             <height>549</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="pushButton_20">
                 <property name="text">
                  <string>Upcoming Events</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_21">
                 <property name="text">
                  <string>Rescheduled Events</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QLabel" name="DatePublished_Events_7">
              <property name="text">
               <string>Date Published</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QTableWidget" name="Events_table_7">
              <row>
               <property name="text">
                <string>7:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>8:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>9:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>10:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>11:00 AM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>12:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>1:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>2:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>3:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>4:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>5:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>6:00 PM</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>7:00 PM</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_22">
              <property name="text">
               <string>View Attendance</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>660</y>
        <width>256</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Request Reschedule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>1271</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>140</y>
            <width>1261</width>
            <height>461</height>
           </rect>
          </property>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Student ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Section</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Course</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender/Sex</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Attendance Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time IN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time OUT</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>61</y>
            <width>291</width>
            <height>71</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Event: Event Name</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>Filter</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>1170</x>
            <y>600</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Go Back</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
